src/routes/docs.ts

```
@@ -0,0 +1,157 @@
+    return new Response(null, { status: 200, headers });
+  }
+
+  if (req.method !== 'GET') {
+    return new Response('Method not allowed', { status: 405, headers });
+  }
+
+  switch (url.pathname) {
+    case '/api/docs':
+    case '/api/docs/':
+      // Swagger UI interface
+      const specUrl = `${url.origin}/api/docs/openapi.json`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)